lib/gradient/error.ex

```diff
@@ -69,6 +69,7 @@ defmodule Gradient.Error do
   @typep ignore_rule ::
            {:ignore_file, file :: charlist()}
            | {:ignore_kind, ignored_kind()}
+           | {:regex, Regex.t()}
```

Comment:
```suggestion
           | {:ignore_regex, Regex.t()}
```

Let's keep it consistent with naming of the other rules